### ISSUE TITLE
Problem with Authorization configuration

### ISSUE THREAD

**Issue by developer-devPHP** [zfcampus/zf-apigility-admin-ui#127]
With latest version of  apigility-admin-ui (version 1.3.7) 
The Authorization creation during interface is incorrect. When via admin panel I am setting Authorization for example for GET method, in module config generating something like this

```php
'zf-mvc-auth' => array(
        'authorization' => array(
            'testapi-V1-Rpc-Testasd-Controller' => array(
                'actions' => array(
                    'testapi-V1-Rpc-Testasd-Controller' => array(
                        'GET' => true,
                        'POST' => false,
                        'PUT' => false,
                        'PATCH' => false,
                        'DELETE' => false,
                    ),
                ),
            ),
        ),
    ),
```

Autorization not working, because the correct structure should be this

```php
 'zf-mvc-auth' => array(
        'authorization' => array(
            'testapi\\V1\\Rpc\\Testasd\\Controller' => array(
                'actions' => array(
                    'testasd' => array(
                        'GET' => true,
                        'POST' => false,
                        'PUT' => false,
                        'PATCH' => false,
                        'DELETE' => false,
                    ),
                ),
            ),
        ),
    ),
```

If I manually changed to above structure Authorization working. 

**Note:**
In version 1.2.4 it worked

**Comment by developer-devPHP** [comment]
#126

**Comment by developer-devPHP** [comment]
@weierophinney What you think about this issue. Can this will fix sooner?

**Comment by jguittard** [comment]
Fixed by #128
